### ISSUE TITLE
Error in scv.tl.recover_dynamics and scv.tl.velocity_graph: The requested array has an inhomogeneous shape after 1 dimensions.

### ISSUE THREAD

**Issue by bsgilgi** [theislab/scvelo#1084]
Hello,
Thank you for the great package.

I am running scvelo after pre-processing with Seurat. The code worked in the past with scvelo 0.2.5 and python 3.8.15.
Currently, I have scvelo 0.2.5 python 3.11.3. 
I get an error when running scv.tl.recover_dynamics or scv.tl.velocity_graph. 

The error is obtained also with the example data and code:
https://scvelo.readthedocs.io/en/stable/vignettes/NatureBiotechCover/

...

Can you please help to solve this?

Thanks a lot

```python
import scvelo as scv
scv.settings.verbosity = 0  # only show errors, no hints/information
scv.set_figure_params('scvelo')  # for beautified visualization
scv.logging.print_version()
adata = scv.datasets.pancreas()
scv.pp.filter_and_normalize(adata, min_shared_counts=30, n_top_genes=2000)
scv.pp.moments(adata)
scv.tl.recover_dynamics(adata)
```

<details>

<summary> Error output </summary>

```pytb
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[8], line 1
----> 1 scv.tl.recover_dynamics(adata)

File ~/workspace/conda_envs/scvelo_jupyter/lib/python3.11/site-packages/scvelo/tools/dynamical_model.py:588, in recover_dynamics(data, var_names, n_top_genes, max_iter, assignment_mode, t_max, fit_time, fit_scaling, fit_steady_states, fit_connected_states, fit_basal_transcription, use_raw, load_pars, return_model, plot_results, steady_state_prior, add_key, copy, n_jobs, backend, **kwargs)
    585     adata.varm["loss"] = loss
    587 if t_max is not False:
--> 588     dm = align_dynamics(adata, t_max=t_max, dm=dm, idx=idx)
    590 logg.info("    finished", time=True, end=" " if settings.verbosity > 2 else "\n")
    591 logg.hint(
    592     "added \n"
    593     f"    '{add_key}_pars', "
    594     f"fitted parameters for splicing dynamics (adata.var)"
    595 )

File ~/workspace/conda_envs/scvelo_jupyter/lib/python3.11/site-packages/scvelo/tools/dynamical_model.py:721, in align_dynamics(data, t_max, dm, idx, mode, remove_outliers, copy)
    718 if dm is not None and dm.recoverable:
    719     dm.m = m[idx]
    720     dm.alpha, dm.beta, dm.gamma, dm.pars[:3] = (
--> 721         np.array([dm.alpha, dm.beta, dm.gamma, dm.pars[:3]]) / dm.m[-1]
    722     )
    723     dm.t, dm.tau, dm.t_, dm.pars[4] = (
    724         np.array([dm.t, dm.tau, dm.t_, dm.pars[4]]) * dm.m[-1]
    725     )
    727 return adata if copy else dm

ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (4,) + inhomogeneous part.
```

</details>

<details> <summary> Versions </summary>

```pytb
scvelo==0.2.5  scanpy==1.9.3  anndata==0.9.1  loompy==3.0.7  numpy==1.24.3  scipy==1.10.1  matplotlib==3.7.1  sklearn==1.2.2  pandas==2.0.2  
```

</details>

**Comment by fingerstylesyf** [comment]
您好，请问问题解决了么，我也遇到这个问题，是版本的原因么

**Comment by Streisenberg** [comment]
@fingerstylesyf I had similar issue and downgrade the numpy to 1.23.5 solved my issue as mentioned at #1044
